jobs/JGLOBAL_ANALYSIS_STATS

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "anlstat" -c "base anlstat"
+
+##############################################
+# Set variables used in the script
+##############################################
+
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+# Generate COM variables from templates
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_OBS COM_CHEM_ANALYSIS
```

Comment:
Workflow will need these modified to be `COMIN` not `COM`, also I suggest a cleaner declaration so that we can add each 'component' on a new line.